src/components/Main.js

```diff
@@ -96,10 +96,17 @@ function injectCSS (url) {
   link.rel = 'stylesheet';
   link.media = 'screen,print';
   link.setAttribute('data-aframe-inspector', 'style');
-  document.getElementsByTagName('head')[0].appendChild(link);
+  document.head.appendChild(link);
 }
 
 (function init () {
+  var webFont = document.createElement('script');
+  webFont.src = 'https://ajax.googleapis.com/ajax/libs/webfont/1.6.16/webfont.js';
+  var webFontLoader = document.createElement('script');
+  webFontLoader.innerHTML = 'WebFont.load({google: {families: ["Roboto Mono"]}});';
+  document.head.appendChild(webFont);
+  document.head.appendChild(webFontLoader);
```

Comment:
Using `document.createElement` + `appendChild` to create and inject a `<script>` together result in the creation, injection, and loading of a script asynchronously.

I haven't tested this pull request locally, but I'm almost positive that you have to wait for the `webFont` script to be created and injected before you inject the `webFontLoader` script.

You should make the following changes, and things should work just fine:

```js
var webFont = document.createElement('script');
webFont.src = 'https://ajax.googleapis.com/ajax/libs/webfont/1.6.16/webfont.js';
var webFontLoader = document.createElement('script');
webFontLoader.innerHTML = 'WebFont.load({google: {families: ["Roboto Mono"]}});';
document.head.appendChild(webFont);
document.head.appendChild(webFontLoader);
```

to

```js
var webFontLoader = document.createElement('script');
webFontLoader.innerHTML = 'WebFont.load({google: {families: ["Roboto Mono"]}});';

var webFont = document.createElement('script');
webFont.src = 'https://ajax.googleapis.com/ajax/libs/webfont/1.6.16/webfont.js';
webFont.addEventListener('load', function () {
  document.head.appendChild(webFontLoader);  
});
webFont.addEventListener('error', function () {
  console.warn('Could not load WebFont script:', webFont.src);
});
document.head.appendChild(webFont);
```